src/Roslyn.Suppressions/Roslyn.Suppressions/InProcess/RoslynSettingsFileSynchronizer.cs

```diff
@@ -115,7 +115,7 @@ public async Task UpdateFileStorageAsync()
             {
                 await threadHandling.SwitchToBackgroundThread();
 
-                var sonarProjectKey = activeSolutionBoundTracker.CurrentConfiguration.Project?.ProjectKey;
+                var sonarProjectKey = configurationProvider.GetConfiguration().Project?.ProjectKey;
 
```

Comment:
The config provider was a straightforward swap in the product and the tests.
This line is the only other change.